addons/sourcemod/configs/vsh/vsh.cfg

```diff
@@ -819,9 +819,8 @@
 		
 		"1104"	//Air Strike
 		{
-			"desp"			"Air Strike: {positive}Adds 1 clip size per 200 damage done, +20% explosion radius"
+			"desp"			"Air Strike: {positive}Adds 1 clip size per 200 damage done"
 			"tags"			"damage_head_required ; 200"
```

Comment:
while were at it, this line can be removed aswell, `tags` key has been unused for years